P/Pfapack/build_tarballs.jl

```diff
@@ -17,15 +17,27 @@ cd $WORKSPACE/srcdir
 cd Pfapack
 
 cmake fortran \
+    -DCMAKE_POSITION_INDEPENDENT_CODE=ON \
     -DCMAKE_INSTALL_PREFIX=${prefix} \
     -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
     -DCMAKE_BUILD_TYPE=Release
 make -j${nproc} VERBOSE=1
 make install
 
+case ${target} in
+
+    *"apple"*)
+        # Apple implements a different linker option.
+        export A2SO="-Wl,-force_load,${prefix}/lib/libpfapack.a"
+        ;;
+    *)
+        export A2SO="-Wl,--whole-archive ${prefix}/lib/libpfapack.a -Wl,--no-whole-archive"
+        ;; 
+esac
```

Comment:
`flagon` does this for you: https://github.com/JuliaPackaging/Yggdrasil/pull/4784#discussion_r853849184

Reply:
I see. Thanks!

Reply:
If you're curious, this is the script: https://github.com/JuliaPackaging/Yggdrasil/blob/acb37f70728e86b5f747f752dbab71a1ce198fb3/0_RootFS/Rootfs/bundled/utils/flagon

Reply:
Impressive. Thanks.